andela/ah-scorpion-frontend
# #160086377 Configure Airbnb linting

**What does this PR do?**

- Change the file format of the `eslintrc` file from `yml` to `json` for easier customization.
- Configure Airbnb linting to accept JSX inside `.js` files.
- Customize the app for Authors Haven.

**How can this PR be manually tested?**

- Clone this project by running `git clone https://github.com/andela/ah-scorpion-frontend.git`.
- Checkout to this branch by running `git checkout ch-configure-airbnb-160086377`.
- Run `npm install` to install dependencies.
- Run `npm run test` to run the existing test. The test should pass.
- Run `npm start` and you should see the message `Welcome to Authors Haven` on the webpage.
- When you open the App.js file in your text editor, you should **NOT** get an eslint warning for having JSX inside files with the `.js` file extension.

**What are the relevant pivotal tracker stories?**

[#160086377](https://www.pivotaltracker.com/story/show/160086377)
